input/templates/partials/common/meta.json

```diff
@@ -1,5 +1,6 @@
 {
 	"assetsPath": "assets/",
 	"bagItems" : [ 1, 2, 3, 4, 5, 6, 7, 8, 9 ],
```

Comment:
I think this does not belong to meta and also the key is not expressive enough.

Sth like `addToCartAllowedItemCounts`?